adafruit_gps.py

```diff
@@ -115,15 +118,15 @@ def send_command(self, command, add_checksum=True):
         Note you should NOT add the leading $ and trailing * to the command
         as they will automatically be added!
         """
-        self._uart.write('$')
+        self._uart.write(b'$')
         self._uart.write(command)
         if add_checksum:
             checksum = 0
             for char in command:
-                checksum ^= ord(char)
-            self._uart.write('*')
-            self._uart.write('{:02x}'.format(checksum).upper())
-        self._uart.write('\r\n')
+                checksum ^= char
+            self._uart.write(b'*')
+            self._uart.write(bytes('{:02x}'.format(checksum).upper(), "utf-8"))
```

Comment:
`"utf-8"` -> `"ascii"`, per previous comment

Reply:
yah im into it. lemme do that